secrets/portworx-with-vault.md

```diff
@@ -39,12 +39,49 @@ If you already have a running Portworx installation, [update `/etc/pwx/config.js
 ### Portworx vault environment variables
 - `VAULT_ADDR=<vault-address>` **Required:** It would be used to connect to the Vault endpoint
 - `VAULT_TOKEN=<vault-token>` **Required:** This token will be used for authenticating Portworx with Vault
+- `VAULT_BASE_PATH=<portworx-base-path>` The base path under which portworx has access to secrets
+- `VAULT_BACKEND_PATH=<secret-backend-path>` If the v2 secret backend is different than `secrets`
 - `VAULT_CACERT=</etc/pwx/path>`
 - `VAULT_CAPATH=/etc/pwx/path>`
 - `VAULT_CLIENT_CERT=</etc/pwx/path>`
 - `VAULT_CLIENT_KEY=/etc/pwx/path>`
 - `VAULT_TLS_SERVER_NAME=<server-name>`
 
+If Vault is configured strictly with policies then the Vault Token provided to Portworx should follow the below policy
+
+```
+# Read and List capabilities on mount to determine which version of kv backend is supported
+path "sys/mounts/*"
+{
+capabilities = ["read", "list"]
+}
+
+# V1 backends
+# Provide full access to the portworx subkey
+# Provide -> VAULT_BASE_PATH=portworx to PX
+path "secrets/portworx/*"
+{
+capabilities = ["create", "read", "update", "delete", "list"]
+}
+
+# V2 backends (Using default backend )
+# Provide full access to the data/portworx subkey
+# Provide -> VAULT_BASE_PATH=portworx to PX
+path "secrets/data/portworx/*"
+{
+capabilities = ["create", "read", "update", "delete", "list"]
+}
+
+# V2 backends (Using custom backend )
```

Comment:
add a V1 example as well

you think we should give an example without base path as well?